tests/cupyx_tests/scipy_tests/stats_tests/test_morestats.py

```diff
@@ -0,0 +1,115 @@
+import warnings
+
+import cupy
+
+from cupy import testing
+
+import cupyx
+import cupyx.scipy.stats  # NOQA
+
+import scipy.stats  # NOQA
+
+
+atol = {
+    'default': 1e-6,
+    cupy.float16: 5e-2,
+    cupy.float32: 1e-5,
+    cupy.float64: 1e-14
+}
+rtol = {
+    'default': 1e-6,
+    cupy.float16: 5e-2,
+    cupy.float32: 1e-5,
+    cupy.float64: 1e-14
+}
+
+
+@testing.with_requires('scipy')
+class TestBoxcox_llf:
+
+    @testing.for_all_dtypes(no_float16=True, no_bool=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', rtol=1e-6)
+    def test_array_1dim(self, xp, scp, dtype):
+        data = testing.shaped_random((2,), xp, dtype=dtype, scale=9)
```

Comment:
Would you use a bit larger input that has 10 elements or so?

Reply:
I see `float16` is not happy with it. I was getting:
```python
E       AssertionError: 
E       Not equal to tolerance rtol=0.005, atol=0
E       
E       x and y -inf location mismatch:
E        x: array(-21.588972)
E        y: array(-inf)
```
So, for this case, I've used 1-dimensional inputs of size 5.